x-pack/legacy/plugins/ml/public/util/context_utils.tsx

```diff
@@ -0,0 +1,66 @@
+/*
```

Comment:
maybe this file should be called `navigation_context` or something like that?
Do we need a place to keep contexts like this? For the wizards i'm currently using the `kibanaContext` from data frames which i think should be moved to a common location at some point.
I'd also be in favour of making `navigation_context` a folder, which would allow the mocks in this file to be moved out.

as we're relying more and more on contexts, we could even just have a `contexts` folder in the root of `public`?

e.g. something like:
```
/public/contexts/navigation_context/
/public/contexts/navigation_context/index.ts
/public/contexts/navigation_context/navigation_context.ts
/public/contexts/navigation_context/mocks.ts
```

Reply:
Great suggestions! As discussed, will do in a follow up and created an issue to track progress #41763.